src/lights/AmbientLight.cpp

```
@@ -0,0 +1,29 @@
+#include "AmbientLight.hpp"
+#include "Point3D.hpp"
+#include "Vector3D.hpp"
+
+Math::Vector3D Raytracer::AmbientLight::computeLighting(
```

Cppcheck (2.10-2)

[style] The function 'computeLighting' is never used.

(unusedFunction)